packages/outdated/src/outdated.ts

```diff
@@ -74,6 +74,10 @@ export default async function outdated (
             return
           }
 
+          if ((opts.manifest.pnpm?.update?.ignore ?? []).includes(alias)) {
```

Comment:
If I understand correctly, this way the ignore field will be per package.json. This is not how the props from the pnpm field are intended to be used. Those setting are only active in the root of the workspace.